File: 3.2/mobile/lab1/lab1/lab1_project2_1_lev/app/src/main/java/com/example/lab1_project2_1_lev/MainActivity.java
package com.example.lab1_project2_1_lev;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button subscribeButton = findViewById(R.id.subscribe_button);
        subscribeButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String url = "https://t.me/lev_sergienko";
                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
                startActivity(intent);
            }
        });
    }
}

File: 3.2/mobile/lab1/lab1/lab1_project2_2_lev/app/src/main/java/com/example/lab1_project2_2_lev/MainActivity.kt
package com.example.lab1_project2_2_lev

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val subscribeButton: Button = findViewById(R.id.subscribe_button)
        subscribeButton.setOnClickListener {
            val url = "https://t.me/lev_sergienko"
            val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
            startActivity(intent)
        }
    }
}